docs/UserGuide.adoc

```diff
@@ -27,92 +39,98 @@ AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for mana
 +
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
+.  Download the latest `clubconnect.jar` link:{https://github.com/CS2103JAN2018-W15-B4/main}/releases[here].
+.  Copy the file to the folder that you want to use as the home folder for Club Connect (and the club Database).
+.  Double-click the file to start the app. The GUI (as shown in <<launch-app, Figure 1>>) should appear in a few seconds.
 +
-image::Ui.png[width="790"]
+[[launch-app]]
+[.text-center]
+.Club Connect application on launch
+image::Ui.png[width="550"]
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
+.  Type a command in the command box and press kbd:[Enter] to execute it. +
+e.g. You can type *`help`* and press kbd:[Enter] to open the help window (shown in Figure 2).
++
+[[help-window]]
+[.text-center]
+.Club Connect Help window
+image::help-window.png[height="450"]
++
+.  Some other example commands that you can try are:
 
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com m/A1234567H` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
-* *`exit`* : exits the app
+* *`list`* : lists all members of the club on the left pane
+* *`add`*`n/John Doe p/98765432 e/johnd@example.com m/A1234567H` : adds a member named `John Doe` to your Club Connect members list
+* *`delete`*`3` : deletes the 3rd member shown in the current list
+* *`exit`* : exits the Club Connect app
 
-.  Refer to <<Features>> for details of each command.
+.  Continue to the next section, <<Features>>, for details of each command and its usage.
 
 [[Features]]
 == Features
 
 ====
 *Command Format*
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Words in `UPPER_CASE` are the parameters to be supplied by the user. For example, in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe` or `add n/Jane Doe`.
+* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/President` or as `n/John Doe`.
+* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/Treasurer`, `t/Operations t/Logistics` etc.
```

Comment:
Logistics is not a tag, it's a group. Maybe remove it or change it to something else?

Reply:
Fixed